### ISSUE TITLE
Native Control Support (ViewPager / UICollectionView)

### ISSUE THREAD

**Issue by dhaligas** [johankson/flipper#12]
what about using a ViewPager for android and a UICollectionView to get the native swiping and virtualization out of the box?

**Comment by johankson** [comment]
Yes, that is perfectly fine to do. This started out as an experiment. I was initially going to add some more custom graphical effects but I might just go back to using the "native" controls to do this. The original plan was to learn a little bit more about custom drawing stuff and see if there was a performance limit. And it just turned into a control. :) If you look at the WP version, we use an out of the box panorama control.
